internal/data/internal/slice_generator_main.go

```diff
@@ -0,0 +1,174 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "os"
+
+const copyright = "// Copyright 2020 OpenTelemetry Authors\n" +
+	"//\n" +
+	"// Licensed under the Apache License, Version 2.0 (the \"License\");\n" +
+	"// you may not use this file except in compliance with the License.\n" +
+	"// You may obtain a copy of the License at\n" +
+	"//\n" +
+	"//      http://www.apache.org/licenses/LICENSE-2.0\n" +
+	"//\n" +
+	"// Unless required by applicable law or agreed to in writing, software\n" +
+	"// distributed under the License is distributed on an \"AS IS\" BASIS,\n" +
+	"// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n" +
+	"// See the License for the specific language governing permissions and\n" +
+	"// limitations under the License.\n\n"
+
+const generatedMessage = "// Code generated by slice_generator_main.go. DO NOT EDIT.\n" +
+	"// To regenerate this file run `go run internal/data/internal/slice_generator_main.go`.\n\n"
+
+const packageAndImport = "package data\n" +
+	"\n" +
+	"import (\n" +
+	"	\"sort\"\n\n" +
+	"	otlpmetrics \"github.com/open-telemetry/opentelemetry-proto/gen/go/metrics/v1\"\n" +
+	")\n\n"
+
+const sliceTemplate = "// ${sliceName} logically represents a slice of ${elementName}.\n" +
```

Comment:
Probably better to use backticks for multi-line string declaration.

Reply:
Done.